barretenberg/cpp/src/barretenberg/polynomials/univariate.test.cpp

```diff
@@ -2,30 +2,21 @@
 #include "barretenberg/ecc/curves/bn254/fr.hpp"
 #include <gtest/gtest.h>
 
-namespace bb::test_univariate {
-
 template <typename FF> class UnivariateTest : public testing::Test {
   public:
-    template <size_t view_length> using UnivariateView = UnivariateView<FF, view_length>;
+    template <size_t view_length> using UnivariateView = bb::UnivariateView<bb::fr, view_length>;
```

Comment:
This is a typed test right? Don't we need `FF` to be a template parameter here? I'd rather leave as-is or make it a non-templated test with fixtures and keep the agnostic `FF` as an alias in the class that can more easily be switched out for another field.

Reply:
my bad here